processor/stream/processor.go

```diff
@@ -82,43 +76,21 @@ func (s *srErrorWrapper) Read() (map[string]interface{}, error) {
 	return v, err
 }
 
+type EventModel struct {
+	Name   string
+	Schema *jsonschema.Schema
+	Decode func(interface{}, time.Time, metadata.Metadata) (transform.Transformable, error)
+}
+
 type Processor struct {
-	Tconfig      transform.Config
-	Mconfig      model.Config
+	Models       []EventModel
```

Comment:
How about making this a map, with the key being the model name? Then we can get rid of EventModel.Name.